test/models/git/repository_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+
+module Git
```

Comment:
We can use `ActiveSupport::TestCase` here instead and do:

```ruby
test "lookup commit for good commit"do
  ...
end
```

Reply:
PR opened: https://github.com/exercism/v3-website/pull/387